multishard_mutation_query.hh

```diff
@@ -67,4 +71,5 @@ future<std::tuple<foreign_ptr<lw_shared_ptr<reconcilable_result>>, cache_tempera
         const query::read_command& cmd,
         const dht::partition_range_vector& ranges,
         tracing::trace_state_ptr trace_state,
-        db::timeout_clock::time_point timeout);
+        db::timeout_clock::time_point timeout,
+        multishard_mutation_query_config cfg);
```

Comment:
Can't we get this from the `db` inside the function, like we do with the other query config stuff?

Reply:
In fact I don't see where this is used, below you do just that.

Reply:
Ugh. I'll drop it.

Reply:
It is wrong to pick it up from db, since we don't know if it's a system or user query. I'll add it to read_command and pick it up on the coordinator side.

Reply:
Yes, that is better, all replicas participating in the read should use the same limit.